### ISSUE TITLE
Settings in about:config?

### ISSUE THREAD

**Issue by RRoenn** [cleidigh/printing-tools-ng#208]
Hello @cleidigh,

I found different keys in my configuration

extensions.printingtoolsng.* and 
extensions.printingtools.* without **ng**

Maybe it is better to merge them together with a new version? 

![grafik](https://user-images.githubusercontent.com/77779094/232476584-6a9630bb-d472-49a1-aff3-1775c963b212.png)

![grafik](https://user-images.githubusercontent.com/77779094/232476699-5772b4fa-b66b-4bc9-a2ff-55e586bbd8df.png)

Greetings
@RRoenn

**Comment by cleidigh** [comment]
@RRoenn 
Whoa... That's bad and I can't believe that happened,
I have to fix this, thanks for catching that!
@cleidigh

**Comment by cleidigh** [comment]
@RRoenn 
Am I correct that you have been a user for a long time, before I even took over PrintingTools?
I just checked, the prefs.js file that defines each pref is 100% 
extensions.printingtoolsng.xyz
Each of the prefs you highlighted were as expected. I also checked the code, all ng...
So i am wondering if these could be from ancient times?

Here is a test:
Try changing header order and some of the others. Then see what changes in about config. Note, close the config page and reopen as it seems to not always refresh.
These have to be old, the code is all consistent 
very weird.
@cleidigh
